config/routes.rb

```diff
@@ -6,6 +6,7 @@
       get "/counties", to: "counties#index"
       get "/county", to: "counties#show"
       get "/trail", to: "trails#show"
+      get '/trails', to: "trails#index"
```

Comment:
It's typically more conventional to put the id for a resource (i.e. the trail or county id) in the path rather than relying on it to be present as a query parameter, which is how it's set up here. 
Instead of line 7 it would be `get 'county/:id', to: 'counties#show'`

Reply:
You typically want to allow the client to mess things up as little as possible (but always still prepare for them to mess things up). By putting the ID in the path, it's harder for them to make a successful request without including an ID, whereas with this current set up, they can make a request without an ID very easily